add docs on query page (#7066)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/navigate-query-page.md

---

> title: "Navigate the Query page interface"

Reviewer: Just in general, we should be ready to update the screenshots this week!

description: "Learn how to navigate the Query page interface"
sidebar_label: "Navigation interface"
tags: [Query page]
---

# Navigate the Query page interface <Lifecycle status="beta" />

<IntroText>
Learn how to navigate the Query page interface and use the main components.
</IntroText>

:::tip

> Available in private beta. To join, please reach out to your account manager. To provide feedback, use this feedback form…

Reviewer: ```suggestion
Available in private beta. To join, please reach out to your account manager. 
```

:::

The Query page provides an interactive interface for writing, running, and analyzing SQL queries. This section highlights the main components of the Query page. 

### Query console
- The Query editor is the main component of the Query page. It allows you to write, run, and analyze SQL queries

> - The Query editor supports syntax highlighting, code completion, and linting.

Reviewer: ```suggestion
- The Query editor supports syntax highlighting and autocomplete suggestions
```

Reviewer: no linting just yet!

> - The Query editor also supports asset linking, which allows you to link to specific tables, columns, and metrics in your project.

Reviewer: I don't know what this means! let's remove

Author: hyperlinking basically! removed!

<Lightbox src="/img/docs/collaborate/query-page/qp-main.png" title="Query page main interface with blank query editor" />

> ### Ellipsis (`...`), Save and Run buttons

Reviewer: Since there's a kebab / ellipsis menu on the results pane too, should we come up with a more general name for this set of buttons?

Reviewer: maybe even just a H2 Query Console --> H3 Query Console Menu situation

- Click on the **ellipsis** button to share, develop in the [IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud), edit details, or delete query.

> - Use the **Save** button to save frequently used SQL queries that you'll often run.

Reviewer: ```suggestion
- Use the **Save** button to save frequently used SQL queries.
```

- Use the **Run** button to run your SQL query and view the results in the **Results** tab

<Lightbox src="/img/docs/collaborate/query-page/qp-ellipsis.jpg" title="Query page ellipsis button" />

### Results, Details, and Chart tabs

> These tabs help you analyze query execution and visualize results

Reviewer: ```suggestion
Use these tabs to analyze query execution and visualize results
```

> - **Results** tab: Displays (or previews) your SQL results in a detailed views. Results are limited to 500 rows, support for more rows coming soon!

Reviewer: ```suggestion
- **Results** tab: Preview your SQL results. Results are paginated.
```

- **Details** tab: Generates succinct details of executed SQL query:

>   - Query metadata: Title, description, supplied SQL, compiled SQL

Reviewer: TItle and description are Copilot generated! would love to highlight that

---
title: "Navigate the Query page interface"
description: "Learn how to navigate the Query page interface"
sidebar_label: "Navigation interface"
tags: [Query page]
image: /img/docs/query-page/qp-results.jpg
---

# Navigate the Query page interface <Lifecycle status="beta,enterprise" />

<IntroText>
Learn how to navigate the Query page interface and use the main components.
</IntroText>

:::tip
Query page is available in private beta to Enterprise accounts. To join, please reach out to your account manager.
:::

The Query page provides an interactive interface for writing, running, and analyzing SQL queries. This section highlights the main components of the Query page. 

## Query console
The query console is the main component of the Query page. It allows you to write, run, and analyze SQL queries. The Query console supports:
- Query console editor, which allows you to write, run, and analyze SQL queries:
  - It supports syntax highlighting and autocomplete suggestions 
  - Hyperlink from SQL code `ref` to the corresponding Explorer page

> - [Query console menu](#query-console-menu), which contains **Bookmark**and **Run** buttons. 

Reviewer: ```suggestion
- [Query console menu](#query-console-menu), which contains **Bookmark** and **Run** buttons. 
```

>   - ✨ Coming soon: Ability to access the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or [Visual Editor](/docs/cloud/visual-editor).

Reviewer: ```suggestion
  - ✨ Coming soon: Ability to open the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or [Visual Editor](/docs/cloud/visual-editor) to continue editing your SQL query.
```
